src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-statsmodels.py

```diff
@@ -0,0 +1,29 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+hiddenimports=[
+'statsmodels.tsa.statespace._kalman_initialization',
+'statsmodels.tsa.statespace._kalman_filter',
+'statsmodels.tsa.statespace._kalman_smoother',
+'statsmodels.tsa.statespace._representation',
+'statsmodels.tsa.statespace._simulation_smoother',
+'statsmodels.tsa.statespace._statespace',
+'statsmodels.tsa.statespace._tools',
+'statsmodels.tsa.statespace._filters._conventional',
+'statsmodels.tsa.statespace._filters._inversions',
+'statsmodels.tsa.statespace._filters._univariate',
+'statsmodels.tsa.statespace._filters._univariate_diffuse',
+'statsmodels.tsa.statespace._smoothers._alternative',
+'statsmodels.tsa.statespace._smoothers._classical',
+'statsmodels.tsa.statespace._smoothers._conventional',
+'statsmodels.tsa.statespace._smoothers._univariate',
+'statsmodels.tsa.statespace._smoothers._univariate_diffuse']
```

Comment:
Are all these really necessary? As far as I see, with 0.12.2, the problem are missing submodules from `_filters` and `_smoothers` that are referenced from cythoned `_kalman_filter`.

Using `collect_submodules` should take care of those:

```python
from PyInstaller.utils.hooks import collect_submodules
hiddenimports = collect_submodules('statsmodels.tsa.statespace._filters') \
              + collect_submodules('statsmodels.tsa.statespace._smoothers')
```